backend/src/org/commcare/xml/StackOpParser.java

```diff
@@ -17,38 +14,34 @@
 /**
  * @author ctsims
  */
-public class StackOpParser extends ElementParser<StackOperation> {
```

Comment:
When I upgraded to Android Studio 2.2 it started highlighting things that can be made package local (the same as `protected`). My resistance to ignoring lint warnings is very low... I should probably just disable that specific highlighting because it may break FormPlayer imports. In this case we should be fine